sdks/jreleaser-java-sdk-commons/src/main/java/org/jreleaser/sdk/commons/feign/FeignLogger.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2020-2023 The JReleaser authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jreleaser.sdk.commons.feign;
+
+import org.jreleaser.logging.JReleaserLogger;
+
+public class FeignLogger extends feign.Logger {
+
+    private final JReleaserLogger logger;
+
+    public FeignLogger(JReleaserLogger logger) {
+        this.logger = logger;
+    }
+
+    @Override
+    protected void log(String configKey, String format, Object... args) {
+        logger.debug(String.format(methodTag(configKey) + format, args));
```

Comment:
This logic is _copied_ from the Feign SLF4J impl, they use String.format directly, because the template string may not match the target impl's logger (e.g. SLF4F's template string)

JReleaserLogger doesn't have an `ifDebugLevel()` method, so this means the string will be formatted even if the debug logger is disabled.

Reply:
May need to add level guards to `JReleaserLogger` to avoid unused formatting.

Reply:
Switching to JSL's `MessageFormat` instead of SLF4J may be an option as well.